charts/nginx-gateway-fabric/values.yaml

```diff
@@ -1,4 +1,12 @@
 nginxGateway:
+  ## Set of custom annotations for GatewayClass objects, e.g:
+  ## gatewayClassCustomAnnotations:
+  ##  someKey: someValue
+  gatewayClassCustomAnnotations: {}
```

Comment:
I would move this field below `gatewayClassName`, and call it `gatewayClassAnnotations`.

Reply:
@sjberman Sorry, I accidentally closed this PR, so created a new one with fixes according to your comments